packages/web-components/src/components/leadspace-with-search/leadspace-with-search.ts

```diff
@@ -24,6 +24,11 @@ const { prefix, stablePrefix: c4dPrefix } = settings;
  * Leadspace with Search
  *
  * @element c4d-leadspace-with-search
+ * @csspart content layout - the content container - Usage: `c4d-leadspace-with-search::part(content-layout)`
+ * @csspart content layout body - the content container body - Usage: `c4d-leadspace-with-search::part(content-layout-body)`
+ * @csspart search container - the search container - Usage: `c4d-leadspace-with-search::part(search-container)`
+ * @csspart search container inner - the search container inner content - Usage: `c4d-leadspace-with-search::part(search-container-inner)`
+ * @csspart sticky header - the sticky header container - Usage: `c4d-leadspace-with-search::part(sticky-header)`
```

Comment:
Something about the formatting here means that this documentation is not being output to the Docs page for this component. Check others to make consistent. While doing local dev, run `yarn wca` in the `packages/web-components` directory to double check that it's getting generated. For instance, [for Link List](https://ibmdotcom-webcomponents.s3.us-east.cloud-object-storage.appdomain.cloud/deploy-previews/11863/index.html?path=/docs/components-link-list--default):

![image](https://github.com/carbon-design-system/carbon-for-ibm-dotcom/assets/191049/0a76ba8f-ed6a-42ae-bc93-66f42bf2c6d6)